src/controls/animatedStatusBar.ts

```diff
@@ -3,14 +3,14 @@
  *--------------------------------------------------------*/
 
 import {
-    StatusBarItem,
+    Disposable,
     StatusBarAlignment,
```

Comment:
Alpha sorting 😍

Reply:
Yup, that is one of the things that TSLINT is picky about.

Reply:
it's beautiful 😍